client/src/app/project-create-edit/components/accession-field-group/accession-field-group.component.spec.ts

```diff
@@ -193,8 +212,21 @@ describe('AccessionFieldGroupComponent', () => {
       expect(geoSeriesCtrl.value).toEqual([]);
       expect(bioStudiesCtrl.value).toEqual([]);
       expect(insdcProjCtrl.value).toEqual([]);
+      expect(dbgapCtrl.value).toEqual([]);
+    });
+    it('should set the DBGAP accession field', () => {
```

Comment:
The proper styling is `dbGaP`, right?
```suggestion
    it('should set the dbGaP accession field', () => {
```